PIM/Operation/04_ManageView.md

```diff
@@ -0,0 +1,179 @@
+
+2. Select the desired channel in the *Channel* drop-down list.   
+  The multi-channel values in the columns have been changed to the values in the selected channel. All single channel values remain unchanged.
+
+  > [Info]  If a product has no attribute value defined in the selected language, the value in the corresponding column remains blank.
+
+
+
+## Select a product category
+
+You can select a product category to limit the number of products to those products that are assigned to a certain category. You can assign a product to one or multiple categories and catalogs in the product view.
+
+#### Prerequisites
+
+- At least one catalog has been created, see [Create a catalog](../Integration/06_ManageCatalogs.md##create-a-catalog).
+- At least one category has been created within the catalog, see [Create a category](../Integration/06_ManageCatalogs.md##create-a-category).
+- At least one product has been assigned to a category, see [Assign a product to a catalog category](../Integration/06_ManageCatalogs.md##assign-a-product-to-a-catalog-category)
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Click the *Catalog* drop-down list in the left column.  
+  All available catalogs are displayed in the list.   
+
+  > [Info] For detailed information about creating a catalog or categories, see [Manage the catalogs](../Integration/06_ManageCatalogs.md).
+
+2. Select the desired catalog in the *Catalog* drop-down list.      
+  All corresponding categories and sub-categories are displayed below the selected catalog.
+
+3. Select the desired category or sub-category in the left column.   
+  Only those products that are assigned to the selected category or sub-category are displayed in the list of products on the right side.
+
+  ![Select category](../../Assets/Screenshots/PIM/Products/List/SelectCategory.png "[Select category]")
+
+  > [Info] When you create a product, it will be automatically assigned to the currently selected category.
+
+
+
+## Select a variant view
+
+Switch the variant view to either display only the master products and single products in the product list or all products including their variants.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+*PIM > Products > Tab LIST*
+
+![Product list](../../Assets/Screenshots/PIM/Products/List/Products.png "[Product list]")
+
+1. Click the *Variants* drop-down list at the top of the product list.   
+  The options below are displayed:
+  - **All products**: All products and their variants are displayed in the product list.   
+  - **Hide variants**: No variants are displayed in the product list.
+
+
+2. Select the desired variant view in the *Variants* drop-down list.
+  Depending on the selection, the product list displays all products or only the master products.
```

Comment:
```suggestion
  Depending on the selection, the product list displays all products or only the master and single products.
```
vgl. oben: Switch the variant view to either display only the master products and single products in the product list or all products including their variants.